lib/net/client.fz

```diff
@@ -42,9 +42,8 @@ public client(f net.family.val, p net.protocol.val, host String, port u16) outco
   s := net.socket_type.by_protocol p
   fuzion.sys.net.connect f.as_num s.as_num p.as_num host port
     .bind net.connection desc->
-      c := (net.connection desc)
-      net.connections.add c
-      c
+      net.connections.add desc
+      net.connection desc
```

Comment:
Yes, would be possible but this makes the code a bit simpler, since connection does not need to implement `property.orderable` and those descriptor numbers are never accessible from outside the module anyway.